src/components/dialogs/filter/expert/expert-filter-utils.ts

```diff
@@ -134,14 +186,31 @@ export const queryValidator: QueryValidator = (query) => {
     const result: ValidationMap = {};
 
     const validateRule = (rule: RuleType) => {
-        const isNumberInput = getDataType(rule.field) === DataType.NUMBER;
-        const isStringInput = getDataType(rule.field) === DataType.STRING;
+        const isValueAnArray = Array.isArray(rule.value);
+        const isNumberInput =
+            getDataType(rule.field) === DataType.NUMBER && !isValueAnArray;
+        const isStringInput =
+            getDataType(rule.field) === DataType.STRING && !isValueAnArray;
         if (rule.id && rule.operator === OperatorType.EXISTS) {
             // In the case of EXISTS operator, because we do not have a second value to evaluate, we force a valid result.
             result[rule.id] = {
                 valid: true,
                 reasons: undefined,
             };
+        } else if (
+            rule.id &&
+            rule.operator === 'between' &&
```

Comment:
Could use rule.operator === OPERATOR_OPTIONS.BETWEEN.name to avoid string hard code?

Reply:
Ah yes, I did for test and forgot to change it